wca-regulations.md

```diff
@@ -208,8 +208,8 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
         - 7f1d) Surface: The flat surface on which the Stackmat has been placed. The mat is considered a part of the surface. The timer is not considered a part of the surface.
     - 7f2) The Stackmat timer must be attached to the mat and placed on the surface, with the timer on the side of the mat nearest to the competitor.
         - 7f2a) Exception: For Solving With Feet, the Stackmat must be placed directly on the floor. The timer device may be placed on the side of the mat farthest from the competitor.
-- 7h) The competition area must have a Competitors Area.
-    - 7h1) The organization team may require that a competitor who has been called to compete must remain within the Competitors Area until the competitor has finished all attempts for the round.
+- 7h) The competition area must have one or more Competitor Waiting Areas.
+    - 7h1) The organization team may require that a competitor who has been called to compete must remain within an assigned Competitor Waiting Area between attempts, until the competitor has finished all attempts for the round.
```

Comment:
> ... within **the** assigned ...?

Each competitor will have one and only one assigned waiting area for every round they compete at.